src/web/components/Card/Card.tsx

```diff
@@ -146,9 +147,21 @@ export const Card = ({
                                         pillar={pillar}
                                         size={headlineSize}
                                         showQuotes={showQuotes}
-                                        kickerText={kickerText}
-                                        showPulsingDot={showPulsingDot}
-                                        showSlash={showSlash}
+                                        kickerText={
+                                            designType === 'Live'
+                                                ? 'Live'
+                                                : kickerText
+                                        }
+                                        showPulsingDot={
+                                            designType === 'Live' && isLiveBlog
+                                                ? true
+                                                : showPulsingDot
+                                        }
+                                        showSlash={
+                                            designType === 'Live' && isLiveBlog
```

Comment:
Nitpick
My preference might be `designType === 'Live' || showSlash` here (if isLiveBlog isn't needed, but actually maybe if it is) - do you prefer the explicit ternary here?

Reply:
As per my other comment, isLiveBlog isn't needed so neither is the ternary (stylistically speaking) and we can use a simple conditional